aries_vcx/src/handlers/connection/invitee/state_machine.rs

```diff
@@ -369,13 +368,13 @@ impl SmConnectionInvitee {
         Ok(Self { state, ..self })
     }
 
-    pub fn handle_problem_report(self, problem_report: ProblemReport) -> VcxResult<Self> {
+    pub fn handle_problem_report(self, _problem_report: ProblemReport) -> VcxResult<Self> {
```

Comment:
I was about to say I strongly oppose not storing problem report after it's received - only to find out we didn't do it previously either and just ignored it in `impl From<(InvitedState, ProblemReport)> for InitialState {` :-) 

So it's good that this modification pointed that out - I think we should store it, what do you think?

Reply:
I am not so sure. Maybe we can start storing it when we have a use case for it?